features/install.feature

```diff
@@ -1,15 +1,19 @@
-Feature: install
+Feature: Installation
 
-  Scenario:
-    Given I am in a node project
-    When I run "mycha install" and respond to the prompt with "spec/spec_helper.coffee"
+  As a developer adding testing to my project
+  I want that the infrastructure required by Mycha is added for me
+  So that I can start testing quickly and don't waste my time following repetitive and boring installation instructions.
```

Comment:
:+1:

I always favor minimal and "without wasting time" is not really needed to me
```
So that I can start testing without repetitive setup work
```